packages/server/graphql/types/SetPhaseFocusPayload.ts

```diff
@@ -18,9 +19,9 @@ const SetPhaseFocusPayload = new GraphQLObjectType<any, GQLContext>({
     },
     reflectPhase: {
       type: new GraphQLNonNull(ReflectPhase),
-      resolve: async ({meetingId}, _args, {dataLoader}) => {
+      resolve: async ({meetingId}, _args: unknown, {dataLoader}) => {
         const meeting = await dataLoader.get('newMeetings').load(meetingId)
```

Comment:
nit: 

```ts
const meeting = (await dataLoader
          .get('newMeetings')
          .load(meetingId)) as MeetingRetrospective
```

will make the error down below go away (not sure why type unions give this error in typescript anyways).